css/components/admin/dataset/form.scss

```diff
@@ -0,0 +1,11 @@
+.c-datasets-form {
+
+  .subscribable-row {
```

Comment:
This is wrong. Look for Metadata form and field Sources. You will see that you only need to set the button margin-top => https://github.com/resource-watch/resource-watch/blame/feature/subscribable-dataset/css/components/admin/dataset/source-item.scss#L1-L6